src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -61,35 +61,35 @@ const buildQuery = ({ useTomeSync }) => {
   const nodeContentFragments = useTomeSync
     ? ''
     : `
-  ${fragments}
-  ${landingPage}
-  ${page}
-  ${healthCareRegionPage}
-  ${healthCareLocalFacilityPage}
-  ${healthCareRegionDetailPage}
-  ${pressReleasePage}
-  ${vamcOperatingStatusAndAlerts}
-  ${newsStoryPage}
-  ${eventPage}
-  ${officePage}
-  ${bioPage}
-  ${vaFormPage}
-  ${benefitListingPage}
-  ${eventListingPage}
-  ${storyListingPage}
-  ${leadershipListingPage}
-  ${healthServicesListingPage}
-  ${pressReleasesListingPage}
-  ${locationListingPage}
-  ${qaPage}
-  ${faqMultipleQaPage}
-  ${stepByStepPage}
-  ${mediaListImages}
-  ${checklistPage}
-  ${mediaListVideos}
-  ${supportResourcesDetailPage}
-  ${basicLandingPage}
-  ${nodeCampaignLandingPage}
+  ${ALL_FRAGMENTS}
+  ${landingPage.fragment}
+  ${page.fragment}
+  ${healthCareRegionPage.fragment}
+  ${healthCareLocalFacilityPage.fragment}
+  ${healthCareRegionDetailPage.fragment}
+  ${pressReleasePage.fragment}
+  ${vamcOperatingStatusAndAlerts.fragment}
+  ${newsStoryPage.fragment}
+  ${eventPage.fragment}
+  ${officePage.fragment}
+  ${bioPage.fragment}
+  ${vaFormPage.fragment}
+  ${benefitListingPage.fragment}
+  ${eventListingPage.fragment}
+  ${storyListingPage.fragment}
+  ${leadershipListingPage.fragment}
+  ${healthServicesListingPage.fragment}
+  ${pressReleasesListingPage.fragment}
+  ${locationListingPage.fragment}
+  ${qaPage.fragment}
+  ${faqMultipleQaPage.fragment}
+  ${stepByStepPage.fragment}
+  ${mediaListImages.fragment}
+  ${checklistPage.fragment}
+  ${mediaListVideos.fragment}
+  ${supportResourcesDetailPage.fragment}
+  ${basicLandingPage.fragment}
+  ${nodeCampaignLandingPage.fragment}
```

Comment:
Each module exporting an addition `.fragment` is what makes these changes backwards-compatible...the same pattern is used by the preview server